[BUG] Cannot run test in debug mode on Visual Studio Code February 2025 (version 1.98)

Hi,

**Describe the bug**
I have updated my Visual Studio Code to version 1.98 and I have experienced that I cannot run a test case in debub mode. I have tried with doing a clean installation of Visual Studio Code version 1.98 and I have faced with same issue. Uninstalling version 1.98 and installing version 1.97 back solved my issue. So perhaps vscode-jest is not yet compatible in some way with 1.98.

**To Reproduce**
Steps to reproduce the behavior:
1. Go to one of your test file
2. Right click on test options
3. Choose Debug Test
4. See error

**Expected behavior**
I would expect that Debug Test is functional as on earlier Visual Studio Code versions.

**Screenshots**

![Image](https://github.com/user-attachments/assets/8eff319b-f8ba-45d9-bfe9-58e31628358c)

**Environment (please complete the following information):**
- vscode-jest version: 6.4.0
- `node -v`: v22.11.0
- `npm -v`: 10.9.0
- jest version: 29.7.0
- your vscode-jest setting: nothing specific
- Operating system: Windows 11

**Prerequisite**
Nx is being used as central tool to launch tests for specific projects.

- are you able to run jest from the command line? Yes, I'm able to do.
- where do you run jest CLI from? From the root of the project
- how do you run your tests from the command line? npx nx run xxxx:test, it launched the Jest CLI with specific arguments defined in the Nx project.json and nx.json

**Additional context**
I have not seen this issue before and unfortunately It does not give me any meaningful information regarding the exact problem.

Same. I tried the `jest.jestCommandLine` and noticed a different behaviour from running the tests and debugging the tests. 

If I set `jest.jestCommandLine": "npx jest"` the tests are running as normal, but once I try to debug it fails with this:

```
cd d:\\source\\workspace ; /usr/bin/env 'NODE_OPTIONS= --require "c:/Users/anonymous/AppData/Local/Programs/Microsoft VS Code/resources/app/extensions/ms-vscode.js-debug/src/bootloader.js"  --inspect-publish-uid=http' 'VSCODE_INSPECTOR_OPTIONS=:::{"inspectorIpc":"\\\\.\\pipe\\node-cdp.xyz.sock","deferredMode":false,"waitForDebugger":"","execPath":"C:\\Program Files\\nodejs\\node.exe","onlyEntrypoint":false,"autoAttachMode":"always","fileCallback":"C:\\Users\\anonymous\\AppData\\Local\\Temp\\node-debug-callback-xyz"}' C:\\Program\ Files\\nodejs\\node.exe .\\npx jest --runInBand --watchAll=false --testNamePattern route\ /my tests\$ --runTestsByPath D:\\source\\workspace\\src\\test\\mytest.test.ts
Debugger attached.
Waiting for the debugger to disconnect...
node:internal/modules/cjs/loader:1228
  throw err;
  ^

Error: Cannot find module 'D:\source\worksace\npx'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1225:15)
```
Notice the part `C:\\Program\ Files\\nodejs\\node.exe .\\npx jest --runInBand`

--- a/src/helpers.ts
+++ b/src/helpers.ts
@@ -314,7 +314,7 @@ export const parseCmdLine = (cmdLine: string): string[] => {
   const parts = cmdLine.match(cmdSplitRegex) || [];
   // clean up command
   if (parts.length > 0 && parts[0]) {
-    parts[0] = removeSurroundingQuote(normalize(parts[0]));
+    parts[0] = normalize(removeSurroundingQuote(parts[0]));
   }
   return parts;
 };

--- a/tests/helpers.test.ts
+++ b/tests/helpers.test.ts
@@ -390,6 +390,8 @@ describe('platform-specific tests', () => {
       ${true}  | ${'\\absolute\\jest --runInBand'}                             | ${{ cmd: '\\absolute\\jest', args: ['--runInBand'], program: '\\absolute\\jest' }}
       ${true}  | ${'"\\dir with space\\jest" --arg1=1 --arg2 2 "some string"'} | ${{ cmd: '\\dir with space\\jest', args: ['--arg1=1', '--arg2', '2', '"some string"'], program: '\\dir with space\\jest' }}
       ${true}  | ${'c:\\jest --arg1 "escaped \\"this\\" string" --arg2 2'}     | ${{ cmd: 'c:\\jest', args: ['--arg1', '"escaped \\"this\\" string"', '--arg2', '2'], program: 'c:\\jest' }}
+      ${true}  | ${'"c:\\jest" --arg1 "escaped \\"this\\" string" --arg2 2'}   | ${{ cmd: 'c:\\jest', args: ['--arg1', '"escaped \\"this\\" string"', '--arg2', '2'], program: 'c:\\jest' }}
+      ${true}  | ${`'c:\\jest' --arg1 "escaped \\"this\\" string" --arg2 2`}   | ${{ cmd: 'c:\\jest', args: ['--arg1', '"escaped \\"this\\" string"', '--arg2', '2'], program: 'c:\\jest' }}
     `('$cmdLine', ({ cmdLine, expected, isWin32 }) => {
       if (!canRunTest(isWin32)) {
         return;

